### ISSUE TITLE
Error from some countries

### ISSUE THREAD

**Issue by PepperAddict** [gwg-women/gwg-women-techmakers#170]
I get this if I put my geolocation from countries that I think google is having trouble recognizing. I think we need a catch to deal with this type of problem.  I couldn't find info from my my mother land: Cambodia :(

![error](https://user-images.githubusercontent.com/4171505/38318113-a009b29e-37fc-11e8-9a5f-d016fdec6be5.png)

**Comment by resant18** [comment]
@PepperAddict, what are the steps to reproduce the error?

**Comment by tanyagupta** [comment]
Hi Pepper 
Can you send me this code (cut and paste)? I can debug it
Thanks
Tanya

**Comment by tanyagupta** [comment]
See if this helps ?
https://codepen.io/tanyagupta/pen/zWJVWx

**Comment by PepperAddict** [comment]
To replicate the error, I used this chrome addon:  https://chrome.google.com/webstore/detail/manual-geolocation/jpiefjlgcjmciajdcinaejedejjfjgki?hl=en

Most countries are okay. Just play around with it and you'll see. 

@tanyagupta I haven't messed with the file yet, but I'll keep that in mind. It could even be something else involved with this issue. Maybe it's actually the addon's fault. *shrugs* I'll take a look at it later.

**Comment by digilou** [comment]
Open another issue if new problems arise after #186 fix.
